### PR TITLE
ollama code coverge

### PR DESCRIPTION
**Pull request by aaronpowell**
Contributing to #168

- **Adding test for data volume name**
- **More tests on the volume mounting**

### CODE REVIEW

**Comment by aaronpowell** [comment]
@Alirexaa I can't figure out how to get the Ollama tests to above 60% (admittedly part of it is the hook which is **really** hard to test), so this is probably as good a coverage as we can get (but at least it's a total over 80% now!).

**Comment by Alirexaa** [comment]
> @Alirexaa I can't figure out how to get the Ollama tests to above 60% (admittedly part of it is the hook which is **really** hard to test), so this is probably as good a coverage as we can get (but at least it's a total over 80% now!).

I think adding funtional test to ensure data volume presistance work could help us. I will take a look at it and try to add more funtional tests.

File: src/CommunityToolkit.Aspire.Hosting.Ollama/CommunityToolkit.Aspire.Hosting.Ollama.csproj [hunk 1]
```diff
@@ -13,4 +13,9 @@
     <Compile Include="..\VolumeNameGenerator.cs" Link="Utils\VolumeNameGenerator.cs" />
   </ItemGroup>
 
+  <ItemGroup>
+    <AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleTo">
+      <_Parameter1>CommunityToolkit.Aspire.Hosting.Ollama.Tests</_Parameter1>
+    </AssemblyAttribute>
```
Comment (Alirexaa) [inline src/CommunityToolkit.Aspire.Hosting.Ollama/CommunityToolkit.Aspire.Hosting.Ollama.csproj:19]:
```suggestion
    <InternalsVisibleTo Include="CommunityToolkit.Aspire.Hosting.Ollama.Tests" ></InternalsVisibleTo>
```

File: tests/CommunityToolkit.Aspire.Hosting.Ollama.Tests/OllamaFunctionalTests.cs
Comment (Alirexaa) [inline tests/CommunityToolkit.Aspire.Hosting.Ollama.Tests/OllamaFunctionalTests.cs:1]: We have a test that verifies the `WithDataVolume` method on the Ollama resource persist data ((`OllamaFunctionalTests.WithDataShouldPersistStateBetweenUsages`)).

We can add the same test for the Open Web UI resource.

**Review by Alirexaa — APPROVED** [review]

File: tests/CommunityToolkit.Aspire.Hosting.Ollama.Tests/OllamaFunctionalTests.cs
Comment (Alirexaa) [reply tests/CommunityToolkit.Aspire.Hosting.Ollama.Tests/OllamaFunctionalTests.cs:1]: @aaronpowell, Adding functional test for Open Web UI data persistence can be follow up.

**Pull request merged**